packages/@orbit/store/test/cache/operation-processors/schema-validation-processor-test.ts

```diff
@@ -0,0 +1,129 @@
+import {
+  KeyMap,
+  Schema,
+  SchemaSettings,
+  ModelNotFound
+} from '@orbit/data';
+import Cache from '../../../src/cache';
+import SchemaValidationProcessor from '../../../src/cache/operation-processors/schema-validation-processor';
+import '../../test-helper';
+
+const { module, test } = QUnit;
+
+module('SchemaValidationProcessor', function(hooks) {
+  let schema, cache, processor;
+
+  const schemaDefinition: SchemaSettings = {
+    models: {
+      node: {
+        attributes: {
+          name: { type: 'string' },
+        },
+        relationships: {
+          parent: { type: 'hasOne', model: 'node', inverse: 'children' },
+          children: { type: 'hasMany', model: 'node', inverse: 'parent' }
+        }
+      }
+    }
+  };
+
+  const node = { type: 'node', id: '1' };
```

Comment:
perhaps name this type `known` instead of `node` (for consistency with `unknown` below)

Reply:
this is a really minor nit - I'm fine with `node` now that I've focused on the relationships you've defined above

Reply:
Yeah, defining relationships are YAGNI at the moment since we don't validate them, but it just felt natural to include them.